### PR TITLE
Feature/comments merge

### PR DESCRIPTION
**Pull request by VladMartyshevskyi**
1. Moved comments handling from editors to ECMS.
2. Reorganized configuration files related to the editors.
3. Small refactoring

### CODE REVIEW

File: apps/portlet-editors/src/main/webapp/WEB-INF/conf/editors/configuration.xml [hunk 1]
```diff
@@ -3,10 +3,6 @@
   xsi:schemaLocation="http://www.exoplatform.org/xml/ns/kernel_1_3.xsd http://www.exoplatform.org/xml/ns/kernel_1_3.xsd"
   xmlns="http://www.exoplatform.org/xml/ns/kernel_1_3.xsd">
 
-  <component>
-    <type>org.exoplatform.wcm.connector.collaboration.DocumentEditorsRESTService</type>
-  </component>
-
   <external-component-plugins>
     <target-component>org.exoplatform.commons.addons.AddOnService</target-component>
```
Comment (pnedonosko) [inline apps/portlet-editors/src/main/webapp/WEB-INF/conf/editors/configuration.xml:7]: This should go to ecm-wcm-extension as wcm-extension/wcm configuration file like this done for app center there.
Finally /conf/editors/configuration.xml  should not exist here.

Comment (VladMartyshevskyi) [reply apps/portlet-editors/src/main/webapp/WEB-INF/conf/editors/configuration.xml]: Moved: https://github.com/exodev/ecms/pull/607/commits/ad86d0fa86039c6bf4d45215bed151b0ab38fcb7

File: core/services/src/main/java/org/exoplatform/services/wcm/utils/WCMCoreUtils.java [hunk 2]
```diff
@@ -732,6 +733,34 @@ public static String getPortalName() {
     PortalContainerInfo containerInfo = WCMCoreUtils.getService(PortalContainerInfo.class) ;
     return containerInfo.getContainerName() ;
   }
+  
+  public static String getCurrentPortalName() throws Exception {
+    UserPortalConfigService userPortalConfigService = WCMCoreUtils.getService(UserPortalConfigService.class) ;
```
Comment (pnedonosko) [inline core/services/src/main/java/org/exoplatform/services/wcm/utils/WCMCoreUtils.java:738]: Get this component only when you need it (below) - save CPU bits :)

Comment (VladMartyshevskyi) [reply core/services/src/main/java/org/exoplatform/services/wcm/utils/WCMCoreUtils.java]: Fixed: https://github.com/exodev/ecms/pull/607/commits/b06976efc6d1a19f56d6ea8f92e749f7805f12fc

**Pull request merged**